src/modules/Contributors/index.jsx

```diff
@@ -16,58 +17,109 @@ export default class Contributors extends Component {
-                    key={login}
-                    className={cx(
-                      "putainde-Contributor",
-                      "r-Tooltip",
-                      "r-Tooltip--bottom",
-                      "r-Tooltip--allowNewLines"
-                    )}
-                    data-r-tooltip={
-                      `${login}\n(${fileContributors[login]} commit${
-                        fileContributors[login] > 1 ? "s" : ""
-                      })`
-                    }
-                  >
-                    <Avatar
-                      author={login}
-                      className="putainde-Contributor-avatar"
-                    />
-                  </div>
+
+    const nbFileContributors = fileContributors
+      ? Object.keys(fileContributors).length
+      : 1
+
+    return (
+      <div className="putainde-Contributors">
+        <strong className="putainde-Contributors-label">
+        {
+          nbFileContributors === 1 &&
+          i18n.BeTheFirstToContribute
+        }
+        {
+          nbFileContributors > 1 &&
+          `${nbFileContributors} ${this.context.i18n.contributors} ` +
+          this.context.i18n.onThisPage
+        }
+        </strong>
+
+        <div className="r-Grid putainde-Contributors-actions">
+          <div className="r-Grid-cell r-all--1of3">
+            <a
+              className="putainde-Contributors-action"
+              href={
+                Contributors.getGithubUrl(
+                  pkg.repositoryHttpUrl,
+                  "edit",
+                  filename
                 )
-              })
-            }
+              }
+            >
+              {i18n.pageEdit}
+            </a>
           </div>
-        )
-      }
-    }
+          <div className="r-Grid-cell r-all--1of3">
+            <a
+              className="putainde-Contributors-action"
+              href={
+                Contributors.getGithubUrl(
+                  pkg.repositoryHttpUrl,
+                  "blame",
+                  filename
+                )
+              }
+            >
+              {i18n.pageBlame}
+            </a>
+          </div>
+          <div className="r-Grid-cell r-all--1of3">
+            <a
+              className="putainde-Contributors-action"
+              href={
+                Contributors.getGithubUrl(
+                  pkg.repositoryHttpUrl,
+                  "commits",
+                  filename
+                )
+              }
+            >
+              {i18n.pageHistory}
+            </a>
+          </div>
+        </div>
 
-    return (
-      <div
-        className={cx(
-          "putainde-Contributors",
-          "putainde-Contributors--noAdditionalContributors"
-        )}
-      >
+        {
+          nbFileContributors > 1 && fileContributors &&
+          Object.keys(fileContributors).map(login => {
+            if (login === "undefined") {
+              console.warn(`${filename} have an undefined contributor.`)
```

Comment:
true say. I will do that (even if it's out of the scope of this pr, the diff is related to an indent change)